docs/HACKING.md

```diff
@@ -174,118 +158,98 @@ Then, in another window
+* grunt lintAndTest: lint and test only (no watch).
 
-Code coverage reports can be created through the `tox -e cover` command:
-
-```bash
-$ tox -e cover
-GLOB sdist-make: /Users/kpepple/Documents/dev/Solinea/goldstone-ui/setup.py
-cover inst-nodeps: /Users/kpepple/Documents/dev/Solinea/goldstone-ui/.tox/dist/goldstone-ui-2014.1.dev56.g0558e73.zip
-cover runtests: commands[0] | coverage run --source=./goldstone manage.py test goldstone --settings=goldstone.settings.local_test
-Creating test database for alias 'default'...
-.........
-----------------------------------------------------------------------
-Ran 9 tests in 0.074s
-
-
-OK
-Destroying test database for alias 'default'...
-cover runtests: commands[1] | coverage xml
-cover runtests: commands[2] | coverage report
-Name                                           Stmts   Miss  Cover
-------------------------------------------------------------------
-goldstone/__init__                                 0      0   100%
-goldstone/apps/__init__                            0      0   100%
-goldstone/apps/lease/__init__                      0      0   100%
-goldstone/apps/lease/admin                         1      0   100%
-goldstone/apps/lease/celery                        3      3     0%
-goldstone/apps/lease/migrations/0001_initial      18      3    83%
-goldstone/apps/lease/migrations/__init__           0      0   100%
-goldstone/apps/lease/models                       34      3    91%
-goldstone/apps/lease/tasks                        21     21     0%
-goldstone/apps/lease/tests                        77      0   100%
-goldstone/apps/lease/tests_celery                 10      0   100%
-goldstone/apps/lease/views                         7      4    43%
-goldstone/libs/__init__                            0      0   100%
-goldstone/settings                                 0      0   100%
-goldstone/settings/__init__                        0      0   100%
-goldstone/settings/base                           24      3    88%
-goldstone/settings/development                     7      7     0%
-goldstone/settings/production                      1      1     0%
-goldstone/settings/stage                           1      1     0%
-goldstone/settings/test                            2      0   100%
-goldstone/urls                                     4      0   100%
-goldstone/wsgi                                     4      4     0%
-------------------------------------------------------------------
-TOTAL                                            214     50    77%
-_______________________________________ summary ___________________
-cover: commands succeeded
-congratulations :)
-```
+As the JavaScript files are concatenated and read from a common file `bundle.js`, you will need to make sure the `grunt watch` task is live and running in order to see changes to JavaScript files reflected in the client.
 
 
-### Front-end testing
+## Managing Goldstone Server Addons
```

Comment:
The installation instructions for the OpenTrail, Leases, and the base add-on project also need to be updated with this, after this is checked in.